server/src/test/java/org/opensearch/cluster/decommission/DecommissionControllerTests.java

```diff
@@ -265,60 +262,6 @@ public void onFailure(Exception e) {
         assertEquals(decommissionAttributeMetadata.status(), newStatus);
     }
 
-    private static class AdjustConfigurationForExclusions implements ClusterStateObserver.Listener {
```

Comment:
This was unused after the previous PR got merged related to atomic update. Removing it in this PR